doc/source/train/examples/deepspeed/gptj_deepspeed_fine_tuning.ipynb

```diff
@@ -413,6 +415,8 @@
     "        \"fp16\": {\n",
     "            \"enabled\": \"auto\",\n",
     "            \"initial_scale_power\": 8,\n",
+    "            \"hysteresis\": 4,\n",
```

Comment:
What's the behavior of `hysteresis` and `consecutive_hysteresis`?

Reply:
Updated the PR description:

> * `hysteresis=4`: You must have 4 batches in a row give NaN/inf gradients before reducing the loss scale factor by 2
> * `consecutive_hysteresis=True`: Any non-NaN/inf gradient batch will reset the counter back to 4.

Reply:
I see. So does that means, if we don't set `hysteresis`, deepspeed will try to scale loss factor by 2 whenever it encounters a batch with nan/inf gradient. 
After we set these params, it will skip the batch with Nan/inf gradients 4 times before rescale the loss?

Reply:
Exactly, we require 4 consecutive NaN batches in order to decrease the loss scale by 1/2. This basically skips more batches in the long run, but results in training that doesn't immediately error out due to getting to the minimum loss scale. Definitely not ideal though.